custom/icds_reports/ucr/data_sources/ccs_record_cases.json

```diff
@@ -1089,7 +1089,73 @@
 		"column_id": "is_availing",
 		"is_nullable": true,
 		"datatype": "integer"
-	 }
+	 },
+     {
```

Comment:
Nit: this looks like it's using a different indentation

Reply:
oh man. I looked into this, it is tabs vs spaces, which is why I had not caught it in my editor. Tempted to leave it to avoid reindenting the whole file, but happy to change if you prefer